openstack/keystone/templates/deployment-api.yaml

```
@@ -231,6 +236,10 @@ spec:
           secret:
             secretName: {{ .Release.Name }}-secrets
             defaultMode: 0444
+        - name: keystone-federation
+          secret:
+            secretName: keystone-federation
```

```suggestion
            secretName: {{ .Release.Name }}-federation
```

To make it match how Secret is named

---

I don't understand why it matters, especially if we hardcode the names everywhere else. Maybe we could remove the `{{ .Release.Name }}` for the secrets above to make it be the same way as everything else?

---

ah, but their name is generated this way... ok then!